npm-shrinkwrap.json

```diff
@@ -1148,9 +1148,9 @@
             }
         },
         "commander": {
-            "version": "2.17.1",
-            "resolved": "https://registry.npmjs.org/commander/-/commander-2.17.1.tgz",
-            "integrity": "sha512-wPMUt6FnH2yzG95SA6mzjQOEKUU3aLaDEmzs1ti+1E9h+CsrZghRlqEM/EJ4KscsQVG8uNN4uVreUeT8+drlgg==",
+            "version": "2.20.0",
```

Comment:
Ugh. I hate this file. Did you update our dependencies or did you just regenerate this file and discover that it was out of date.

Keeping dependencies up to date is important, but I'd rather not do it in the same PR as this change.  If you just made that one change to package.json and followed the steps to regenerate this file, then I suppose it is fine.

Reply:
This is partially a result of #1127 (Resolve high severity `npm audit` vulnerabilities) and partially a result of running `npm install` with a newer npm release.